dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50BasicTests.war/src/io/openliberty/el50/fat/basic/servlets/EL50BasicTestsServlet.java

```diff
@@ -33,6 +38,7 @@
  * Servlet for testing basic changes in EL 5.0
```

Comment:
Can we update this to spell out "Expression Language"?

Reply:
Sure, I'll go through a couple and update them all as I go